### ISSUE TITLE
Minor issue with Linked NPC Name Exclusions

### ISSUE THREAD

**Issue by HitBug83** [Synthesis-Collective/SynthEBD#34]
Hello, not a big deal, but if I add a Linked NPC Name Exclusion, it is not registered for the next patching run.

I have to exit SynthEBD to update the JSON and then it works when I rerun the patcher. It looks like it reads the saved list instead of the actual list.

BTW for those who are using the mod Interesting NPCs, you need to add "Skooma addict" to this list if you want to avoid errors (and ingame an Imperial junkie with a Khajiit skin)

**Comment by Piranha91** [comment]
Thanks! I just uploaded a fixed commit, but since this is a minor issue as you mentioned I'll hold off on making a new release for a day or two to see if anything else comes in so as to avoid spamming. BTW I'm moving the loose json files into their parent settings (linked NPC groups and excluded names -> general settings, trim paths -> texmesh settings). Apologies if you've made further modifications; you'll need to re-make them as it'll re-initialize the defaults. I did add Skooma Addict to the default exclusions. Thanks for the report!
